aea/configurations/schemas/skill-config_schema.json

```diff
@@ -80,6 +78,14 @@
         "$ref": "definitions.json#/definitions/public_id"
       }
     },
+    "customs": {
```

Comment:
Are we making this a required or optional field?

Reply:
Optional